network-api/networkapi/wagtailpages/pagemodels/blog/blog_index.py

```diff
@@ -91,10 +94,20 @@ class BlogIndexPage(IndexPage):
     with additional logic to explore topics.
     """
 
+    related_topics = ParentalManyToManyField(
+        BlogPageTopic,
+        help_text='Which topics would you like to feature on the page? '
+                  'Please select a max of 7.',
+        blank=True,
+        verbose_name='Topics',
```

Comment:
**Question** Why change the heading here? The field is called "related_topics" and is displayed on the page as "Related topics". Not sure what the use of the different name here would be.

Reply:
Good point! Going to remove